app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "sunny@treeni.com"
```

Comment:
replace the email with a more relevant sender. selfassesment@unglobalcompact.org or something.